packages/core/EventManager.ts

```diff
@@ -338,9 +338,18 @@ export default class EventManager {
   private async createAllCalendarEvents(event: CalendarEvent) {
     let createdEvents: EventResult<NewCalendarEventType>[] = [];
     if (event.destinationCalendar && event.destinationCalendar.length > 0) {
-      for (const destination of event.destinationCalendar) {
-        if (destination.credentialId) {
-          let credential = this.calendarCredentials.find((c) => c.id === destination.credentialId);
+      // Want to only create one event per calendar app as they sync across their own platforms
+      const destinationCalendarApps = event.destinationCalendar.reduce((apps, calendar) => {
+        if (!apps.includes(calendar.integration)) apps.push(calendar.integration);
+        return apps;
+      }, [] as string[]);
+      for (const destination of destinationCalendarApps) {
+        const destinationCalendar = event.destinationCalendar.find(
+          (calendar) => calendar.integration === destination
+        );
+        if (destinationCalendar?.credentialId) {
+          // Doesn't matter who creates the event for team events
+          let credential = this.calendarCredentials.find((c) => c.id === destinationCalendar?.credentialId);
```

Comment:
Since this one event was being created per calendar app I don't think it matters too much who's event this is being created on. 

Users shouldn't be changing calendar events on the calendar anyways, changes should be made on the Cal.com booking.